Bio/phenotype/__init__.py

```diff
@@ -201,29 +201,29 @@ def read(handle, format):
     exactly one record.  For example, reading a PM JSON file:
 
     >>> from Bio import phenotype
-    >>> record = phenotype.read("plate.json", "pm-json")
+    >>> record = phenotype.read("phenotype/Plate.json", "pm-json")
     >>> print("ID %s" % record.id)
-    PM09
+    ID PM01
     >>> print("Number of wells %i" % len(record))
     Number of wells 96
 
     If the handle contains no records, or more than one record,
     an exception is raised.  For example:
 
-    >>> from Bio import phenotype
-    >>> record = phenotype.read("plates.csv", "pm-csv")
-    Traceback (most recent call last):
+        from Bio import phenotype
```

Comment:
You need a double colon on the preceding paragraph to make this an RST literal block.